src/tasks/index.js

```diff
@@ -1,3 +1,12 @@
 import studySessionTasks from './studySession';
 
-export default [...studySessionTasks];
+const notifySubscribers = studySessionTasks[0];
+const sendChannelReminder = studySessionTasks[1];
+
+const tasks = {
+    "minute": [],
+    "hour": [notifySubscribers],
+    "five-hour": [sendChannelReminder]
+};
```

Comment:
I love the separation here! It definitely gives us a lot more control. What do you think about having this `sendChannelReminder` function get called when new study sessions/events are made?
Hopefully that would also remove any frustration of users scheduling a session and having to wait a number of hours before it shows up in the events channel.
https://github.com/chris-mcdonald-dev/korean-discord-app/blob/1741c8285d9d1d210a0e1af86b4b681d683c495d/src/scripts/activities/study-session.js#L53

Reply:
That makes a lot of sense! 

In fact, since that would handle all cases where a study session is added to the list (there's no other way to create a study session), the only thing that would be handled by the scheduler is the case when removing a study session from the list (either through cancellation or the date passing), which I think we can get away with being less urgent with an update for - could maybe drop the scheduled task time to 12-hourly? Daily?

Also, I haven't checked but I think once a study session passes it remains in the database forever, do you think those are useful to keep around? If not, I think it would be a "good first issue" to create a weekly/fortnightly/monthly scheduled task that deletes all study sessions that have passed